SimTracker/SiPhase2Digitizer/test/DigiTest_cfg.py

```diff
@@ -64,15 +62,20 @@
 #process.digi_step = cms.Sequence(process.siPixelRawData*process.siPixelDigis)
 process.p = cms.Path(process.digiana_seq * process.dqm_comm )
 
-# customisation of the process.                                                                                                                              
-
-# Automatic addition of the customisation function from SLHCUpgradeSimulations.Configuration.combinedCustoms                                                 
-#from SLHCUpgradeSimulations.Configuration.combinedCustoms import cust_2023tilted
+# customisation of the process.
 
-#call to customisation function cust_2023tilted imported from SLHCUpgradeSimulations.Configuration.combinedCustoms                                          
-#process = cust_2023tilted(process)
 # Automatic addition of the customisation function from SLHCUpgradeSimulations.Configuration.combinedCustoms
-from SLHCUpgradeSimulations.Configuration.combinedCustoms import cust_2023LReco 
+from SLHCUpgradeSimulations.Configuration.combinedCustoms import cust_2023tilted4021 
```

Comment:
this function doesn't exist at this moment- is it coming or is it now removed thanks to the eras migration (which is where this needs to go at some point)